CMakeLists.txt

```diff
@@ -224,7 +224,7 @@ configure_file(src/bootloader/bootloader_version.h.in src/bootloader/bootloader_
 string(APPEND CMAKE_C_FLAGS " -std=c11 -pipe")
 if(CMAKE_CROSSCOMPILING)
   set(CMAKE_C_FLAGS "\
-    ${CMAKE_C_FLAGS} -mcpu=cortex-m4 -mthumb -mlong-calls \
+    ${CMAKE_C_FLAGS} -mcpu=cortex-m4 -mthumb -mlong-calls -mno-unaligned-access \
```

Comment:
Is this safe, or does this introduce risk? If it's safer to not use this flag we could postpone using this trick until we need to save bytes more urgently.

Reply:
Yeah, not sure what the impact has. I think unaligned accesses can be problematic on some platforms, so I would assume that **no** unaligned access would be safer than allowing it.